lib/pbench/server/api/resources/users_api.py

```diff
@@ -1,16 +1,91 @@
 from http import HTTPStatus
 import jwt
-from flask import request, jsonify, make_response
+from flask import request, jsonify, make_response, current_app as flask_app
 from flask_restful import Resource, abort
 from flask_bcrypt import check_password_hash
+from flask_mail import Mail, Message
 from email_validator import EmailNotValidError
 from sqlalchemy.exc import SQLAlchemyError, IntegrityError
 from typing import NamedTuple
+
+from pbench.server.api.resources import (
+    Schema,
+    Parameter,
+    ParamType,
+)
 from pbench.server.database.models.users import User
 from pbench.server.database.models.active_tokens import ActiveTokens
 from pbench.server.api.auth import Auth
 
 
+class ForgotPassword(Resource):
+    def __init__(self, config, logger, auth):
+        self.auth = auth
+        self.config = config
+        self.logger = logger
+        self.mail = Mail(flask_app)
```

Comment:
Another option would have been to pass the app as a constructor parameter along with `config, logger, auth`; but I guess this works.